src/Files.App/Views/PaneHolderPage.xaml

```diff
@@ -34,14 +33,38 @@
 			<SolidColorBrush x:Key="PaneSelectedBorderBrush" Color="{ThemeResource SystemAccentColorLight1}" />
 
 			<mconv:BoolNegationConverter x:Key="BoolNegationConverter" />
+
 		</ResourceDictionary>
 	</Page.Resources>
 
-	<Grid HorizontalAlignment="Stretch" VerticalAlignment="Stretch">
+	<Page.KeyboardAccelerators>
+		<KeyboardAccelerator
+			Key="Left"
+			Invoked="KeyboardAccelerator_Invoked"
+			Modifiers="Control,Shift" />
+		<KeyboardAccelerator
+			Key="Right"
+			Invoked="KeyboardAccelerator_Invoked"
+			Modifiers="Control,Shift" />
+		<KeyboardAccelerator
+			Key="W"
+			Invoked="KeyboardAccelerator_Invoked"
+			Modifiers="Control,Shift" />
+		<KeyboardAccelerator
+			Key="Add"
+			Invoked="KeyboardAccelerator_Invoked"
+			Modifiers="Menu,Shift" />
+		<KeyboardAccelerator
+			Key="{x:Bind local:PaneHolderPage.PlusKey}"
+			Invoked="KeyboardAccelerator_Invoked"
+			Modifiers="Menu,Shift" />
+	</Page.KeyboardAccelerators>
+
+	<Grid x:Name="RootGrid">
 		<Grid.ColumnDefinitions>
 			<ColumnDefinition
 				x:Name="LeftColumn"
-				Width="1.0*"
+				Width="*"
```

Comment:
Why the change?

Reply:
1*, 1.0*, *

All of them is the same and should be constant.